lib/spectrum/holding/mirlyn_item.rb

```diff
@@ -11,7 +11,7 @@ def initialize(holding_input:,item_info:,
         @item = item_factory.call(@id, @raw, item_info)
         @item_info = item_info
       end
-      def to_a(action: Spectrum::Holding::Action.new(@id, @id, @bib_record, @holding, @item_info),
+      def to_a(action: Spectrum::Holding::Action.for(@id, @id, @bib_record, @holding, @item_info),
```

Comment:
The first two params are for "id" and "datastore", and for get_this at least they are the same thing, the doc id. 

It looks like differences in "record" and "datastore" happen for linked records, but not for "Get This" "Book This" or "Request This". Is that about right? 

Basically I'd like to collapse these params if possible.

Reply:
They are *supposed* to be different for GetThis, that is a bug, but Jon has apparently worked around it.  I suspect that it can be collapsed, though.